src/net.h

```diff
@@ -49,6 +50,9 @@ static const int TIMEOUT_INTERVAL = 20 * 60;
 static const unsigned int MAX_INV_SZ = 50000;
 /** The maximum number of new addresses to accumulate before announcing. */
 static const unsigned int MAX_ADDR_TO_SEND = 1000;
+/** The maximum # of bytes to receive at once */
+static const int MAX_RECV_CHUNK = 256*1024;
```

Comment:
No they are different.

RECV_SHAPER_MIN_FRAG is basically saying "if the bucket is limiting us to less the X bytes, then don't bother to read anything at all (because its inefficent to read 1 byte at a time)" -- that is what is the minimum fragment we should read?  It is currently set to 256 bytes."

MAX_RECV_CHUNK is saying "Even if the leaky bucket is allowing us to receive 10MB, only grab X bytes in this loop iteration".  This was a hard-coded value (size of the buffer allocated on the stack) in the original code.  It is set to 256K bytes.